boards/dptechnics/walter/doc/index.rst

```diff
@@ -0,0 +1,284 @@
+.. _walter:
+
+DPTechnics Walter
+#################
+
+Overview
+********
+
+Walter is a compact IoT development board that combines an Espressif ESP32-S3 SoC
+with a Sequans Monarch 2 GM02SP LTE-M/NB-IoT/GNSS modem.
+
+To help you get the most out of Walter, we have published the `schematic`_ and the
+`KiCAD footprint`_, along with our `datasheet`_ which describes all pins and test pads.
```

Comment:
this is the zephyr documentation, not DPTechnics' :) Please refrain from using "we" this way as this is confusing/inappropriate. Please use a more neutral tone to enumerate the list of external resources available to the reader.

Reply:
Thanks for the feedback, we didn't mean to cause confusion here :). Fixed!